docs/index.md

```diff
@@ -6,14 +6,25 @@ title: Tinner
 [![CI Status](https://github.com/AY2122S2-CS2103T-T17-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103T-T17-1/tp/actions)
 [![codecov](https://codecov.io/gh/AY2122S2-CS2103T-T17-1/tp/branch/master/graph/badge.svg?token=HVJ16S6MRB)](https://codecov.io/gh/AY2122S2-CS2103T-T17-1/tp)
 
+Main window  
 ![Ui](images/Ui.png)
 
-**Tinner is a desktop application for managing your internship applications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+Reminder window   
+![Reminder](images/Reminder.png)
+
+Tinner (Anagram of Intern) is a desktop app for managing internship applications for students, optimized for use via a Command Line Interface (CLI)
+while still having the benefits of a Graphical User Interface (GUI).
+If you can type fast, Tinner allows you to easily sort through and retrieve relevant information faster than traditional GUI apps.
+
+Tinner aims to solve the hassle of managing internship applications though 3 aspects.
```

Comment:
As discussed, can consider changing this part to:

`1. **Tracks** essential information such as statuses and deadlines at a glance.
2. **Reminds** you of important deadlines so that you will not miss anything about your application.
3. Allows you to **Review** the process and take down notes so that you can ace your next application.`